drivers/hmis_external_apis/app/models/hmis_external_apis/external_id.rb

```diff
@@ -13,6 +13,7 @@ class ExternalId < GrdaWarehouseBase
     belongs_to :external_request_log, optional: true
     belongs_to :remote_credential, class_name: 'GrdaWarehouse::RemoteCredential'
     belongs_to :source, polymorphic: true
+    belongs_to :client, -> { where(external_ids: { source_type: 'Hmis::Hud::Client' }) }, foreign_key: 'source_id', class_name: 'Hmis::Hud::Client', optional: true
```

Comment:
This is really minor, but I worry about our table names in queries since often they don't match the usual rails conventions (in this case it does.)  Would it work to use Arel, or are we in some special case here?
```suggestion
    belongs_to :client, -> { where(arel_table[:source_type].eq('Hmis::Hud::Client')) }, foreign_key: 'source_id', class_name: 'Hmis::Hud::Client', optional: true
```

Reply:
I don't know if it would work with Arel table like that.

Reply:
I think I may have the wrong table (it's probably a different class' area table we need.) but it _should_ be equivalent.

Reply:
Just pushed something similar which I think is what you were getting at @eanders .

Reply:
Yes! Thanks @yakloinsteak